src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -25,7 +25,7 @@ task buildImage(type: DockerBuildImage, dependsOn: dockerFile) {
         BuildResult result = build('buildImage')
 
         then:
-        !result.standardOutput.contains('Step 1 : FROM ubuntu:12.04')
+        !result.standardOutput.contains('Step 0 : FROM ubuntu:12.04')
```

Comment:
Docker version. docker-java is only reporting on what docker is outputing. I was testing on 1.8.3 which is supported. I think we want to at least support that as docker-java does.